src/components/textarea/textarea.scss

```diff
@@ -1,12 +1,14 @@
 @import 'colors.scss';
+@import 'input-field.scss';
+
 
 .carbon-textarea .common-input__label {
   margin-top: 9px;
   vertical-align: top;
 }
 
 .carbon-textarea__input {
-  height: auto;
```

Comment:
we should not remove the `height: auto` - this is important to ensure Textarea is a proper height. It should only be the same height as other inputs if `expandable` is set I think.

Reply:
So should I set the `height: auto` next to `min-height`? Or cancel the PR and make my changes for the page I need?

Reply:
I think having `height: auto` and `min-height: $input-common-height` should work nicely together 👍

Reply:
@ianoxley @harpalsingh 

So do you guys think the `min-height` should be ` $input-common-height` or `2 *  $input-common-height`?

Reply:
I would think `min-height` should be `$input-common-height` as it needs to sit inline with other inputs when `expandable` prop is set to `true`.